library/std/Cargo.toml

```diff
@@ -139,7 +139,8 @@ test = true
 level = "warn"
 check-cfg = [
     'cfg(bootstrap)',
-    'cfg(target_arch, values("xtensa"))',
+    'cfg(target_arch, values("xtensa", "aarch64-unknown-nto-qnx710_iosock"))',
```

Comment:
...this seems incorrect?

Reply:
This was required to get it compiling and should be removed/reverted once nightly knows about the target. See also https://rustc-dev-guide.rust-lang.org/building/new-target.html?highlight=check-cfg#adding-a-target-specification